src/components/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import icon from "../assets/images/night.png";
+import style from "./Header.module.scss";
+import iconMoon from "../assets/images/moon.png";
+import { useContext } from "react";
+import { CustomContext } from "../helpers/Context";
+
+function Header() {
+  const { isTheme = true, toggleTheme } = useContext(CustomContext);
+  return (
+    <>
+      {isTheme ? (
+        <div className={style.head_block}>
+          <h1>Where in the world?</h1>
+          <h3 onClick={toggleTheme}>
+            <img src={icon} alt="moon"></img>
+            Dark Mode
+          </h3>
+        </div>
+      ) : (
+        <div className={`${style.head_block} ${style.light}`}>
+          <h1>Where in the world?</h1>
```

Comment:
I think you repeat yourself with the markup.

I propose creating a different approach
```js
const C = () => {
  const containerClassnames = `${style.head_block}${isDarkTheme ? ` ${style.light}` : ''`}`;
  const imageAltText = isDarkTheme ? 'dark moon' : 'moon'; // actually this one may be the same text
  const themeText = isDarkTheme ? 'Light theme' : 'Dark theme'
  const ThemeIcon = isDarkTheme ? Icon : IconMoon;

  return <div className={containerClassnames}>
    <h1>Where in the world?</h1>
   <h3 onClick={toggleTheme}>
            <img src={ThemeIcon} alt={imageAltText} />
            {themeText}
    </h3>
 </div>
}
```